storefront-reference-architecture-master/cartridges/int_twilio/cartridge/client/default/js/product/detail.js

```diff
@@ -0,0 +1,131 @@
+var base = require("base/product/detail");
+
+function popUpMessage(messageType, msg) {
+    if ($(".add-to-cart-messages").length === 0) {
+        $("body").append('<div class="add-to-cart-messages"></div>');
```

Comment:
$('body') is requested multiple times. Can be defined as constant

Reply:
fixed